runtime/compiler/control/CompilationThread.cpp

```diff
@@ -498,7 +498,7 @@ bool TR::CompilationInfo::shouldDowngradeCompReq(TR_MethodToBeCompiled *entry)
                  getMethodQueueSize() >= TR::Options::_qszThresholdToDowngradeOptLevelDuringStartup) ||
                  // Downgrade if AOT and startup, 
                 (TR::Options::getCmdLineOptions()->sharedClassCache() &&
-                 _jitConfig->javaVM->phase == J9VM_PHASE_STARTUP &&
```

Comment:
What is the difference between J9VM_PHASE_STARTUP and !J9VM_PHASE_NOT_STARTUP ?

Reply:
From the JVM point of view there are 3 phases, EARLY_STARTUP, STARTUP and NOT_STARTUP.
The condition was changed to include the EARLY_STARTUP phase.